egs2/voxceleb/spk1/conf/tuning/train_ecapa_tdnn.yaml

```diff
@@ -1,44 +1,56 @@
 # ECAPA-TDNN reproduce recipe configuration.
 
-frontend: fbank
+# Frontend
+frontend: default
+frontend_conf:
+    hop_length: 128
+    n_mels: 80
 
+# Normalizer
+normalize: utterance_mvn
+normalize_conf:
+    norm_vars: false
```

Comment:
This is an example of adding normalize function with only mean subtraction on utterance level. We would need some modification in the collect stats stage for global mvn, but since we are unlikely to use it, I just add this one here instead of adding tens of lines in the shell script.